### ISSUE TITLE
Can't open settings - ST4

### ISSUE THREAD

**Issue by luckman212** [redoPop/SublimeGremlins#10]
Hi @redoPop 👋 

I love this little plugin and have been using it for years
I recently switched over to ST4 (4098) and set everything up again from scratch.

Not sure what's wrong, but when I try to go to Package Settings > Gremlins > Settings (or keybinds) it just throws an error.

![image](https://user-images.githubusercontent.com/1992842/110681505-e798bb80-81a7-11eb-82fc-ad7c9a0889c6.png)

In console:
![image](https://user-images.githubusercontent.com/1992842/110679265-46a90100-81a5-11eb-943a-b7c7dc13c2ff.png)

I checked and there is no file there. But, even cloning the repo from Github and manually placing the `Gremlins.sublime-settings` file in my user prefs directory (`~/Library/Application Support/Sublime Text/Packages/User`) it still doesn't find it. 

I'm not sure if ST4 changed the API for plugins to load settings, but thought I would ask...

**Comment by redoPop** [comment]
Thanks for flagging this! I just published v1.1.1 with a fix.

f58d96f4 describes the issue in a little more detail, but the gist is that I was relying on an undocumented package-relative lookup path to load settings files in the package directory. ST4 doesn't have the same behavior and it's unclear to me whether that's a beta bug or just a difference in implementation so I've added a couple of custom commands to smooth over the difference.
